src/core/utils/compatibility.js

```diff
@@ -166,6 +166,13 @@ export function isCompatibleWithUserAgent({
     return { compatible: false, reason: INCOMPATIBLE_NOT_FIREFOX };
   }
 
+  // Fenix does not support add-ons (yet?).
+  // See: https://github.com/mozilla-mobile/fenix/issues/1134
+  // See also: https://github.com/mozilla/addons-frontend/issues/7963
```

Comment:
https://github.com/mozilla/addons-frontend/issues/9864 is probably a better issue to mention?

Reply:
Yeah, that was a copy and paste of the code that was there before the patch that removed it. I'll update it.